packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -38,6 +38,31 @@ const Tolerance _kFlingTolerance = Tolerance(
   distance: 0.01,
 );
 
+/// Configures how an [AnimationController] behaves when animations are disabled.
+///
+/// When [AccessibilityFeatures.disableAnimations] is enabled, animations are
+/// speed up or modified to remove most of the visual component. This enum is
+/// used to allow certain AnimationControllers to opt out of this behavior, for
+/// the purpose of preserving some visual animation or simulation behavior for
+/// accessibility.
+///
+/// For example, the [AnimationController which controls the physics simulation
+/// for a scrollable list will have [AnimationBehavior.unscalable] so that when a
+/// user attempts to scroll it does not jump to the end/beginning too quickly.
+enum AnimationBehavior {
+  /// The [AnimationController] will reduce or remove the visual component by
+  /// adjusting simulation logic.
+  normal,
+
+  /// The [AnimationController] will preserve its behavior.
+  ///
+  /// This is the default for repeating animations in order to prevent them from
```

Comment:
This value is typically used by repeating animations, like indeterminate progress indicators, to prevent the AnimationController from ticking at all?

Also: since (as far as I can tell) this value means that we don't do anything when the device requests that animations be disabled, it would be helpful to say as much.

Reply:
I renamed the enum to "preserve" since this captures the concept better, scaling was more related to the time dilation stuff.